installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,122 @@
+====
+
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster
++
+[IMPORTANT]
+====
+If you have an AWS profile stored on your computer, it must not use a temporary session token that you generated while using a multi-factor authentication device. The cluster continues to use your current AWS credentials to create AWS resources for the entire life of the cluster, so you must use key-based, long-lived credentials. To generate appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You can supply the keys when you run the installation program.
+====
+* You downloaded the AWS CLI and installed it on your computer. See link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or UNIX)] in the AWS documentation.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[manually create and maintain IAM credentials].
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+[id="installation-requirements-user-infra_{context}"]
+== Requirements for a cluster with installer-provisioned infrastructure
+
+For a cluster that contains installer-provisioned infrastructure, the Machine API will be responsible for creating the worker nodes declared by the MachineSets available on `<install-dir>/openshift/99_openshift-cluster-api_worker-machineset-*.yaml`
+
+This section describes the requirements for deploying {product-title} on an existing VPC with Local Zone subnets.
+
+include::modules/installation-machine-requirements.adoc[leveloffset=+2]
+include::modules/installation-minimum-resource-requirements.adoc[leveloffset=+2]
+include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
```

Comment:
The types supported by Local Zone are not the same to this list, only few instance types are available [1] in Local Zone, and the types supported by each Local Zone are also different. At present, only `m5.xlarge` are tested by QE. 
I would suggest remove this section or adding a new list for Local Zone separately. 

@patrickdillon @mtulio WDYT?

[1] https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/?pg=localzones

Reply:
> The types supported by Local Zone are not the same to this list, only few instance types are available [1] in Local Zone, and the types supported by each Local Zone are also different. At present, only `m5.xlarge` are tested by QE. I would suggest remove this section or adding a new list for Local Zone separately.
> 
> @patrickdillon @mtulio WDYT?
> 
> [1] https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/?pg=localzones

@yunjiang29 I agree to add a separate list. I would suggest validating at least three instance types. Some time ago I exported the matrix of instances supported. As the instances can vary between zones, I'd suggest to validated those instances:
- `m5.xlarge`
- `m5.2xlarge`
- `c5d.2xlarge`
- `t3.xlarge`

https://gist.github.com/mtulio/c98aa15128a7becb06a372f00d824c42

Reply:
@mtulio To be clear, `supported` doesn't equal to `tested`, for example, there are tons of instance types in various AWS regions, but we just tested some of them which are listed in the documentation. (that's why we call it `Tested instance types for AWS`, but user can use any instance types which meet [minimum resource requirements](https://docs.openshift.com/container-platform/4.11/installing/installing_aws/installing-aws-customizations.html#installation-minimum-resource-requirements_installing-aws-customizations) )

I will do validation on above instance types, and create PR on installer side to add a new list next Monday, so that this PR can include it. Following the current rule, the list would be:

```
m5.*
c5d.*
t3.*
```

cc @GroceryBoyJr @kalexand-rh

Reply:
For now, I'm making a manual edit to show `m5.xlarge`, since that is true right now, and we can replace the list as soon as that PR clears.